validate_sptx/examples/experiment/no_manifest.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
would suggest naming this no_fov_manifest, as manifest is one of those highly overloaded terms.

Reply:
Very fair. Will push shortly.